### ISSUE TITLE
tensorflow is not a supported wheel on this p latform

### ISSUE THREAD

**Issue by Radzhab** [tensorflow/tensorflow#10061]
I have python 2.7.13 (Anaconda 4). When i try install **tensorflow** its raise error.
```
tensorflow_gpu-1.1.0-cp35-cp35m-win_amd64.whl is not a supported wheel on this p
latform.
```

I try use gpu version - same result.

**Comment by foxik** [comment]
Only Python 3.5 packages are available on Windows -- so use Python 3.5 instead of 2.7.

**Comment by gunan** [comment]
The comment by @foxik is correct.
On windows, TF only has support for Python 3.5
